src/app/grid/grid-save-state/grid-state.component.html

```diff
@@ -33,8 +36,9 @@
     </div>
 
     <igx-grid [id]="gridId" #grid1 igxState [data]="localData" [primaryKey]="'EmployeeID'" width="950px" height="550px"
-        [rowEditable]="true" [allowFiltering]="true" [rowSelectable]="true"
-        [paging]="true" [columnPinning]="true">
+        [rowEditable]="true" [allowFiltering]="true" [allowAdvancedFiltering]="true" [rowSelectable]="true"
+        [paging]="true" [showToolbar]="true" [columnPinning]="true" [columnHiding]="true"
+        [filterMode]="'excelStyleFilter'" [displayDensity]="'compact'">
```

Comment:
`[displayDensity]="'compact'"` looks a bit tiny, could you change that to cosy? You will also need to update the sample height in the topic.

Reply:
Done